src/callbacks_step/amr.jl

```diff
@@ -471,6 +471,65 @@ function (amr_callback::AMRCallback)(u_ode::AbstractVector, mesh::P4estMesh,
     return has_changed
 end
 
+function (amr_callback::AMRCallback)(u_ode::AbstractVector, mesh::T8codeMesh,
```

Comment:
Would it make sense to restrict this to the 
```suggestion
function (amr_callback::AMRCallback)(u_ode::AbstractVector, mesh::SerialT8codeMesh,
```
since it doesn't support MPI right now? Or would it throw an error already earlier?